Book-A-Desk.Api/BookingNotifier.fs

```diff
@@ -90,21 +91,31 @@ module rec BookingNotifier =
             |> mailMessage.Cc.Add
         
         mailMessage.Subject <- emailNotificationDetails.Subject
-        let bodyPart = TextPart("plain")
+        let bodyPart = TextPart(TextFormat.Html)
         
         bodyPart.Text <- emailNotificationDetails.Text
         mailMessage.Body <- bodyPart
         mailMessage
-        
+
+    let healthQuestionnaireMailMessage =
+            $"SVP, veuillez répondre au questionnaire sur l’état de santé. Si vous répondez oui à une des questions, nous vous demandons de rester à la maison.{Environment.NewLine}{Environment.NewLine}" +
```

Comment:
The sentence in French is correct but a more correct/formal way would be:

Merci de bien vouloir répondre au questionnaire sur l’état de santé. Si vous répondez oui à l'une des questions, nous vous demandons de rester chez vous.

Reply:
@steverenardbroadsign this message comes from the existing email sent right now by people in Montreal office. I don't know anything about French! XD

Reply:
Yes, I know but we could have asked them to modify the PDF file. It is a detail but looks so weird for me having a sentence starting with a upper case "SVP" ... :- )

Reply:
Let's set up a meeting for next week and suggest that changes to Karol ;)